docs/CQ.rst

```diff
@@ -0,0 +1,59 @@
+Create a Contribution Questionnaire
+===================================
+
+All the new dependencies and all the dependencies updates should be submited as a new CQ into the
+`ECLIPSE IP Zilla <https://dev.eclipse.org/ipzilla/query.cgi>`__.
+
+Submitting a CQ
+---------------
+
+Visit `https://projects.eclipse.org/projects/locationtech.geotrellis/ <https://projects.eclipse.org/projects/locationtech.geotrellis/>`__
+and log into the system.
+
+.. figure:: img/locationtech-geotrellis.png
+   :alt: The GeoTrellis project page
+
+On the right, in the ``COMMITTER TOOLS`` side bar, you can find a link `Create a Contribution Questionnaire <https://projects.eclipse.org/projects/locationtech.geotrellis/cq/create>`__
+
+.. figure:: img/geotrellis-cq.png
+   :alt: Create a Contribution Questionnaire page
+
+Next, we need to create a Third-Party Code Request usually. All the next steps would be based on a particular library example.
+Let's create a CQ for ``pureconfig v0.10.2``:
+
+.. figure:: img/pb-cq.png
+   :alt: A "piggyback" CQ
+
+If your library of your version is here, then select it. It is the easiest result and it means that somebody have already sent a request to
```

Comment:
"When typing in the name/version of the dependency, it may sometimes already appear. If that's the case, then that means someone else has already sent this dependency to Eclipse's IP team. In that case, the CR could be instantly approved if the IP team has already approved of the given dependency."